### ISSUE TITLE
Testing getters and setters

### ISSUE THREAD

**Issue by MHKEND01** [courtGibson/MyPlanner#10]
UserTest and DeparmentTest ultimately just test getters and setters, not necessary.

**Comment by A-Garuba** [comment]
Correct, you can just have an equals override or toString and test (single method) if you want to get 100% coverage more easily
